resources/aws/routetable.go

```diff
@@ -10,14 +10,14 @@ import (
 )
 
 type RouteTable struct {
```

Comment:
IMO when "route table" are two words, the file should be called `route_table.go`. I know, also a little late here. ✌️